packages/AutoTDD-Tests.package/ATDDBrowserTests.class/class/setTestHasRun.st

```diff
@@ -1,4 +1,4 @@
-as yet unclassified
+accessing
 setTestHasRun
```

Comment:
Should probably be renamed. No `set` and `get` in Squeak

Reply:
not a setter (takes no arguments): set and reset